src/pages/PokemonDetailsPage/PokemonDetailsPage.css

```
@@ -105,3 +101,65 @@
 .bold {
   font-weight: bold;
 }
+
+@media screen and (max-width: 440px) {
+  .button-and-name {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```

you can use [repeat](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat) here to avoid having to manually define all 7 grid columns. 

e.g

```suggestion
    grid-template-columns: repeat(7, 1fr);
```